zapd/models.py

```diff
@@ -161,6 +161,32 @@ def in_status(cls, session, status):
     def __repr__(self):
         return "<Proposal %r>" % (self.id)
 
+class Wallet(db.Model):
+    id = db.Column(db.Integer(), primary_key=True)
+    wallet_address = db.Column(db.String(), unique=True)
+    devices = db.relationship('Device', backref='wallet', lazy=True)
+
+    @classmethod
+    def from_wallet_address(cls, session, wallet_address):
+        return session.query(cls).filter(cls.wallet_address == wallet_address).first()
+
+    def __repr__(self):
+        return "<Wallet Addres %r>" % (self.wallet_address)
+
+class Device(db.Model):
+    id = db.Column(db.Integer(), primary_key=True)
+    device_imei_id = db.Column(db.String(), unique=True)
```

Comment:
_id is probably redundant in device_imei_id

Reply:
changed.